invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsDetails/DocumentItems/DocumentItems.js

```diff
@@ -0,0 +1,247 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import {
+  Accordion, Checkbox, Divider,
+  Grid, Header,
+  Label,
+  List,
+  Menu,
+  Segment
+} from "semantic-ui-react";
+import {
+  getDisplayVal,
+  invenioConfig
+} from "../../../../../common/config/invenioConfig";
+
+class DocumentItem extends Component {
+
+  constructor(props){
+    super(props);
+  }
+
+  render() {
+    const {item} = this.props;
+    return <List.Item key={item.pid}>
+      <List.Icon name={'barcode'} size={'large'} verticalAlign={'middle'}/>
```

Comment:
Maybe it's just me but I would expect to see the actual barcode if I tried to click on a barcode icon. What do you think?

Also, what happens if there are a lot of items? Do we need pagination for them?

Reply:
I didn't do the pagination, I was thinking to wait to validate this feature before I will add complexity, @ntarocco WDYT?